test/fixtures/get-orderbook-response.json

```diff
@@ -324,9 +324,9 @@
           "value": "0.5",
           "counterparty": "rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B"
         },
-        "totalPrice": {
+        "price": {
```

Comment:
One serious problem though is that we only know the quality for orderbook orders, not for offercreate transactions or account orders. We can compute it for offercreate transactions, but I think for account orders it might not be possible to show the correct exchange rate for orders that have been partially executed. Maybe we have to ask rippled to show it? Or do account_offers not get updated after a partial execution?